packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -346,34 +346,6 @@ void main() {
         ProcessManager: () => FakeProcessManager.any(),
       },
     );
-
-    testUsingContext(
-      'legacy analytics disabled will disable new analytics',
-      () async {
-
-        io.setExitFunctionForTests((int exitCode) {});
-
-        await runner.run(
-          <String>[],
-          () => <FlutterCommand>[],
-          // This flutterVersion disables crash reporting.
-          flutterVersion: '[user-branch]/',
-          shutdownHooks: ShutdownHooks(),
-        );
-
-        expect(globals.flutterUsage.enabled, false);
-        expect(globals.analytics.telemetryEnabled, false);
-        expect(testLogger.statusText.contains(
-          'Please note that analytics '
-          'reporting was already disabled'), true);
-      },
-      overrides: <Type, Generator>{
-        Analytics: () => FakeAnalytics(),
-        FileSystem: () => MemoryFileSystem.test(),
-        ProcessManager: () => FakeProcessManager.any(),
-        Usage: () => legacyAnalytics,
-      },
-    );
```

Comment:
This test was no longer necessary since legacy opt out status will be honored from within `package:unified_analytics` for every tool using that package